index.bs

```diff
@@ -315,6 +315,30 @@ For example,
 is more readable than
 <code highlight="js">window.scrollBy(50, 0)</code>.
 
+<h3 id="promises">Use Promises</h3>
```

Comment:
I'd title this "Designing asynchronous APIs" perhaps since a large chunk of it is about the general question of sync-vs.-async.